resources/todomvc/architecture-examples/jquery/package.json

```diff
@@ -1,10 +1,13 @@
 {
   "private": true,
+  "scripts": {
+    "build": "node scripts/build.js"
+  },
   "dependencies": {
-    "director": "~1.2.2",
-    "handlebars": "^4.0.6",
-    "jquery": "^3.1.1",
-    "todomvc-app-css": "^2.1.0",
-    "todomvc-common": "^1.0.4"
+    "director": "^1.2.8",
```

Comment:
Is there data on popularity of these non jquery libraries? The bump to jQuery 3.6.* makes total sense as per the data in https://docs.google.com/document/d/1MXxk9DryJmvAmPMt2uMneaS3J5Y-dMFGt4EGvMzhhJE/edit#heading=h.ajnxpgmazu31, although I have no idea on director & handlebars what is used most frequently.

Reply:
If it would help resolve the discussion in #120 perhaps we could revert the upgrade for these two libs and restrict this PR only to jQuery version bump (to the most commonly used version) + common todo assets + directory cleanup? cc @rniwa

Reply:
sounds good - reverted handlebars and director. 
scores:
https://gist.github.com/flashdesignory/10b78168b0e253eb07576b75bd38b63e

Reply:
That's fine with me - we can update to this more common jQuery version and we'll debug any issues with the score in Firefox. @rniwa what do you think?

Reply:
Yeah, just updating jQuery makes sense to me.

Reply:
awesome - @rniwa  - does that mean this could get approved?

Reply:
Yes!